### ISSUE TITLE
GTX 980M 30FPS

### ISSUE THREAD

**Issue by limtasheng** [ValveSoftware/Dota-2#977]
Hello, i am having this problem since Wednesday, when in game my fps was only 30 + even i set the lowest graphic, high packet loss during in game, when in main menu i experience low fps as well is pretty lag in main menu. Please Help me to solve this issue. Thanks

**Comment by gdrewb-valve** [comment]
Can you post your full Steam system information (in the Steam client under the Help menu)?

**Comment by limtasheng** [comment]
Computer Information:
    Manufacturer:  Notebook  
    Model:  P65_P67RGRERA
    Form Factor: Laptop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x5e
    CPU Stepping:  0x3
    CPU Type:  0x0
    Speed:  2592 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Windows 8.1 (64 bit)
    NTFS:  Supported
    Crypto Provider Codes:  Supported 311 0x0 0x0 0x0

Video Card:
    Driver:  Intel(R) HD Graphics 530

```
DirectX Driver Name:  nvumdshim.dll
Driver Version:  20.19.15.4474
DirectX Driver Version:  10.18.13.6881
Driver Date: 13 June 2016
OpenGL Version: 4.5
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 60 Hz
DirectX Card: NVIDIA GeForce GTX 980M
VendorID:  0x10de
DeviceID:  0x13d7
Revision:  0xa1
Number of Monitors:  1
Number of Logical Video Cards:  1
No SLI or Crossfire Detected
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 1920 x 1080
Primary Display Size: 20.91" x 11.77"  (23.98" diag)
                                        53.1cm x 29.9cm  (60.9cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 1024 MB
Supported MSAA Modes:  2x 4x 8x 
```

Sound card:
    Audio device: Speakers (Realtek High Definiti

Memory:
    RAM:  8009 Mb

Miscellaneous:
    UI Language:  English
    Microphone:  Not set
    Media Type:  Undetermined
    Total Hard Disk Space Available:  1191686 Mb
    Largest Free Hard Disk Block:  210139 Mb
    OS Install Date: Jan 01 1970
    Game Controller: None detected
    VR Headset: None detected

**Comment by limtasheng** [comment]
Here it is.

**Comment by limtasheng** [comment]
Computer Information:
Manufacturer: Notebook

Model: P65_P67RGRERA
Form Factor: Laptop
No Touch Input Detected

Processor Information:
CPU Vendor: GenuineIntel
CPU Family: 0x6
CPU Model: 0x5e
CPU Stepping: 0x3
CPU Type: 0x0
Speed: 2592 Mhz
8 logical processors
4 physical processors
HyperThreading: Supported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Unsupported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Supported

Network Information:
Network Speed:

Operating System Version:
Windows 8.1 (64 bit)
NTFS: Supported
Crypto Provider Codes: Supported 311 0x0 0x0 0x0

Video Card:
Driver: Intel(R) HD Graphics 530

DirectX Driver Name:  nvumdshim.dll
Driver Version:  20.19.15.4474
DirectX Driver Version:  10.18.13.6881
Driver Date: 13 June 2016
OpenGL Version: 4.5
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 60 Hz
DirectX Card: NVIDIA GeForce GTX 980M
VendorID:  0x10de
DeviceID:  0x13d7
Revision:  0xa1
Number of Monitors:  1
Number of Logical Video Cards:  1
No SLI or Crossfire Detected
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 1920 x 1080
Primary Display Size: 20.91" x 11.77"  (23.98" diag)
                                        53.1cm x 29.9cm  (60.9cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 1024 MB
Supported MSAA Modes:  2x 4x 8x

Sound card:
Audio device: Speakers (Realtek High Definiti

Memory:
RAM: 8009 Mb

Miscellaneous:
UI Language: English
Microphone: Not set
Media Type: Undetermined
Total Hard Disk Space Available: 1191686 Mb
Largest Free Hard Disk Block: 210139 Mb
OS Install Date: Jan 01 1970
Game Controller: None detected
VR Headset: None detected

**Comment by gdrewb-valve** [comment]
This forum is for Mac and Linux-only bugs.  You'll want to post at dev.dota2.com
